File: bpcp-frontend/src/app/report/dto/reportDTO.ts
export interface reportDTO {
    id: number;
    movementAt: "string";
    name: "string";
    status: "string";
    numberAccount: "string";
    typeAccount: "string";
    initialAmount: "Double";
    movementAmount: "Double";
    totalAmount: "Double";

    createAt?: Date;
    updateAt?: Date;
    
  }

File: movement/src/main/java/com/bcpc/movement/controller/dto/ReportDTO.java
package com.bcpc.movement.controller.dto;

import lombok.*;

import java.sql.Timestamp;
import java.time.LocalDate;

@Getter
@Setter
@Data
@AllArgsConstructor
@NoArgsConstructor
public class ReportDTO {

    private Timestamp movementAt;
    private String name;
    private String numberAccount;
    private String typeAccount;
    private Integer status;
    private Double initialAmount;
    private Double movementAmount;
    private Double totalAmount;

}